packages/shared/test-suites/RecordingTestMetadata.ts

```diff
@@ -411,8 +411,10 @@ export async function processCypressTestRecording(
         }
 
         const annotations = userActionEventIdToAnnotations[id];
-
-        assert(annotations != null, `Missing annotations for test event (${command.name})`);
+        if (!annotations) {
+          // console.warn(`No annotations: ${id}`, command);
+          return;
+        }
```

Comment:
helpful for seeing what we'd miss if we didn't throw. I dont plan on landing this.